sdk/contentsafety/Azure.AI.ContentSafety/api/Azure.AI.ContentSafety.netstandard2.0.cs

```diff
@@ -70,7 +70,9 @@ public partial class ContentSafetyClient
         public virtual Azure.Response AnalyzeText(Azure.Core.RequestContent content, Azure.RequestContext context = null) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response<Azure.AI.ContentSafety.AnalyzeTextResult>> AnalyzeTextAsync(Azure.AI.ContentSafety.AnalyzeTextOptions body, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response> AnalyzeTextAsync(Azure.Core.RequestContent content, Azure.RequestContext context = null) { throw null; }
+        public virtual Azure.Response<Azure.AI.ContentSafety.TextBlocklist> CreateOrUpdateTextBlocklist(string blocklistName, Azure.AI.ContentSafety.TextBlocklist resource, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
It would be good to check with architects regarding whether the name of the method should start with `CreateOrUpdate` or just `Update`.

Reply:
`createOrUpdateTextBlocklist` is user-defined operation name. @KrzysztofCwalina should we normalize it to `updateTextBlocklist`?